backends/p4fmt/attach.cpp

```diff
@@ -0,0 +1,78 @@
+#include "backends/p4fmt/attach.h"
+
+#include "frontends/common/parser_options.h"
+#include "lib/source_file.h"
+
+namespace P4::P4Fmt {
+
+Attach::~Attach() = default;
```

Comment:
nitpick: I would prefer having a plain default destructor in the .h, or better yet just avoid it. The compiler will, of course generate it. This way it may look like from the .h file like it actually does something (which is suspicious :-D).